README.md

```diff
@@ -403,17 +404,15 @@ public class FidelityCard extends RealmObject {
 ---
 
 
-### Gérer les enum
-
-Certains attributs de type Int peuvent représenter des enum. Pour gérer ce cas, il faut ajouter les deux paires clé-valeur suivantes à **l'attribut** :
+### Managing enums
+
+It can happen that an `Int` attribute represents an enum member. To deal with this case, you will have to add the following two key/value pairs to the **attribute**:
 
 | Key | Value |
-|-----|-------|
-| `enumType` | `mon_type` |
-| `enumValues` | `ma_valeur_1,ma_valeur_2,ma_valeur_3` |
-
-
-__Exemple__: Sur l'attribut 'type' de l'entité 'shop':
+| `enumType` | `my_type` |
+| `enumValues` | `my_value_1, my_value_2, my_value_3` |
```

Comment:
Please keep the `| --- | …` line (that's why it was important to bring your fork up-to-date, that's a new change since your original fork that you'll need to keep).

Without that line, GitHub won't render that markdown as a proper table.